draft-ietf-rats-corim.md

```diff
@@ -1100,24 +1100,85 @@ The `uint` and `text` types MUST NOT be interpreted in a global scope.
 
 #### Reference Values Triple {#sec-comid-triple-refval}
 
-A Reference Values triple relates reference measurements to a Target
-Environment. For Reference Value Claims, the subject identifies a Target
-Environment, the object contains measurements, and the predicate asserts that
-these are the expected (i.e., reference) measurements for the Target
-Environment.
+The Reference Values Triple has the following structure:
 
 ~~~ cddl
 {::include cddl/reference-triple-record.cddl}
 ~~~
 
+The `reference-triple-record` has the following parameters:
+
+* `ref-env`: Search criteria that target Evidence environments in the ACS.
```

Comment:
This seems to be a significant shift in the earlier description of the triple! The triple semantics should not require us to describe where the triple is targeted to internally, in the description of the triple.

It should simply say:

What this triple is about :
What does it convey
i.e it conveys for a target Env identified by the Env: The Reference measurements identified by ref-claims

it should not expain anything about the ACS here

Reply:
Separate comment: I am not sure I know how to parse that sentence. Quick help: what's an "Evidence environment"? I assume I am missing something obvious here.

Reply:
> Separate comment: I am not sure I know how to parse that sentence. Quick help: what's an "Evidence environment"? I assume I am missing something obvious here.

Is it more clear to just say: "Search criteria that targets Evidence."